src/Tizen.Sensor/Tizen.Sensor/BatchData/BatchData.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Sensor.BatchData
+{
+    /// <summary>
+    /// The BatchData class is used for storing value of particular batch type sensor.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public abstract class BatchData
+    {
+        /// <summary>
+        /// Initializes a new instance of the BatchData class.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="timeStamp">Timestamp of sensor event.</param>
+        /// <param name="accuracy">Sensor accuracy of sensor event.</param>
+        public BatchData(TimeSpan timeStamp, SensorDataAccuracy accuracy)
+        {
+            Timestamp = timeStamp;
+            Accuracy = accuracy;
+        }
+
+        /// <summary>
+        /// Gets the value of the HeartRateMonitorBatch Timestamp.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <value> TimeSpan </value>
+        public TimeSpan Timestamp
```

Comment:
TimeSpan not good for represent Timestamp
TimeSpan represents a time interval.

Reply:
@myroot You're right, but It's already being used to represent the timestamp of the type in other sensor implementations. Would you recommend fixing all of this at this opportunity?

Reply:
@myroot Is `DateTime` or `DateTimeOffset` suitable? How do you think?

Reply:
If it is represent a real time and it use local time, use `DateTime`, if it use UTC time, use `DateTimeOffset`, 
if it is not represent a real time and just it represent flows of times, just use `long` type